include/openenclave/enclave.h

```diff
@@ -579,6 +580,100 @@ oe_result_t oe_get_seal_key_v2(
  */
 void oe_free_seal_key(uint8_t* key_buffer, uint8_t* key_info);
 
+/**
+ * Get enclave-specific key information that could be passed to
+ * oe_get_seal_key_v2() to derive a symmetric encryption key coupled to the
+ * enclave platform.
+ *
+ * @param[in] seal_policy The policy for the identity properties used to derive
+ * the seal key.
+ * @param[in] entropy Optional parameter containing additional entropy for key
+ * derivation. If \c NULL, oe_get_seal_key_info() will generate entropy on
+ * behalf of the caller.
+ * @param[in] entropy_size Must be \c 0 if \p entropy is \c NULL.
+ * @param[in] opt_flags \c 0 to use TEE defaults. If \c OE_SEAL_SGX_SPECIFIC is
+ * specified, the rest of \p opt_flags is interpreted as the \a attribute_mask
+ * for \c EGETKEY on SGX.
+ * @param[out] key_info On success this points to the enclave-specific key
+ * information which can be used to retrieve the key by passing it to
+ * oe_get_seal_key_v2(). Freed by calling oe_free_key().
```

Comment:
What is "Freed by calling oe_free_key()" referring to?  The key_info?  If so, it should either be called oe_free_key_info() or just use oe_free(), to avoid confusion with the key returned by get_seal_key.

Reply:
`oe_free_key()` an existing API defined at https://github.com/openenclave/openenclave/blob/master/include/openenclave/enclave.h#L543. It frees either a `key_buffer`, or a `key_info`, or both. Am I missing anything?